src/common/i18n/translations/en.ts

```diff
@@ -86,7 +86,7 @@ export const en: Translations = {
     previouslyRedeemed: "Previously redeemed",
     showMore: "Show more",
     showLess: "Show less",
-    today: "for today",
+    today: " for today",
     limitReachedRecent: "Limit reached %{time} ago %{today}.",
     limitReachedDate: "Limit reached on %{dateTime} %{today}.",
     limitReached: "Limit reached %{today}.",
```

Comment:
Since you already introduced a space in `%{today}`, we can do this:
```suggestion
    limitReachedRecent: "Limit reached %{time} ago%{today}.",
    limitReachedDate: "Limit reached on %{dateTime}%{today}.",
    limitReached: "Limit reached%{today}.",
```

Reply:
@ilhaoni Can I include a space between ago and %{today} ?